src/System.Private.CoreLib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -467,14 +467,14 @@ public virtual IList<CustomAttributeTypedArgument> ConstructorArguments
             }
         }
 
-        public virtual IList<CustomAttributeNamedArgument>? NamedArguments
+        public virtual IList<CustomAttributeNamedArgument> NamedArguments
         {
             get
             {
                 if (m_namedArgs == null)
                 {
                     if (m_namedParams == null)
-                        return null;
+                        return null!;
```

Comment:
this doesn't look obvious but our code shouldn't ever be able to get here. Only time we can get here is if someone derives from this class and doesn't initialize these fields (one of the constructors is protected and doesn't initialize it but non internal code uses it and uses NamedArguments at the same time)